### PR TITLE
feat(subscription): add subscription alias

### PR DESCRIPTION
**Pull request by Red-Asuka**

### CODE REVIEW

**Review by CrazyWisdom — APPROVED** [review]

File: src/types/global.d.ts [hunk 1]
```diff
@@ -107,6 +107,7 @@ declare global {
 
   interface SubscriptionModel {
     topic: string
+    alias: string
```
Comment (ysfscream) [inline src/types/global.d.ts:110]: Should be optional.

File: src/components/SubscriptionsList.vue [hunk 2]
```diff
@@ -84,6 +84,11 @@
               </el-input>
             </el-form-item>
           </el-col>
+          <el-col :span="24">
+            <el-form-item label="Alias">
```
Comment (ysfscream) [inline src/components/SubscriptionsList.vue:88]: Translate the label.

File: src/components/SubscriptionsList.vue [hunk 3]
```diff
@@ -121,6 +126,7 @@ export default class SubscriptionsList extends Vue {
   private showDialog: boolean = false
   private subRecord: SubscriptionModel = {
     topic: 'testtopic/#',
+    alias: '',
```
Comment (ysfscream) [inline src/components/SubscriptionsList.vue:129]: No default value required.

**Review by 0721Betty — APPROVED** [review]

File: src/components/SubscriptionsList.vue [hunk 4]
```diff
@@ -40,7 +40,7 @@
               class="topic"
               @click.stop="stopClick"
             >
-              {{ sub.topic }}
+              {{ sub.alias ? sub.alias : sub.topic }}
```
Comment (ysfscream) [inline src/components/SubscriptionsList.vue:43]:
```js
sub.alias || sub.topic
```

**Review by ysfscream — APPROVED** [review]

**Pull request merged**